### DISCUSSION TITLE
HA stop log dopo cambio ora

### DISCUSSION THREAD

**Discussion by sky7176** [indomus/forum#986] [Hardware & Firmware]
Buona sera a tutti,
anche a voi vi si è bloccato la registrazione del log dopo il cambio d'ora?
Io mi sono trovato tutti i log bloccati sino a che questa mattina non ho riavviato HA.

Giuseppe

**Comment by Moxser** [comment]
Ciao. È stato aperto un issue qui https://github.com/home-assistant/core/issues/58783.
È successa la stessa cosa pure a me questa mattina. Dopo il riavvio tutto è tornato alla normalità.

**Reply by sky7176** [reply]
Grazie
